tests/test_tables/test_records.py

```diff
@@ -20,6 +23,19 @@ def test_table_records_factory(table: Table) -> None:
 
 def test_table_records_lists_factory(table: Table) -> None:
     table_records_lists = records.lists_factory(table)
+    _do_test_table_records_lists_factory(table, table_records_lists)
+
+
+def test_table_records_lists_factory_for_table_without_primary_key() -> None:
+    metadata = MetaData()
+    table = Table('numbers', metadata, Column('number', Integer))
```

Comment:
this can be a fixture in **tests.fixtures.tables** module
```python
@pytest.fixture(scope='function')
def table_without_unique_columns(metadata: MetaData) -> Table:
    return Table('numbers', metadata, Column('number', Integer))
```

Reply:
if we generalize `tables` strategy as described [here](https://github.com/lycantropos/hypothesis_sqlalchemy/pull/9#discussion_r262797773), then there is no need in this fixture